jerry-math/log.c

```diff
@@ -105,7 +105,7 @@ log (double x)
   {
     if (((hx & 0x7fffffff) | lx) == 0) /* log(+-0) = -inf */
     {
-      return -two54 / zero;
+      return -INFINITY;
```

Comment:
How come that line 112 does not have to be edited? That's a div0 as well, only for NaN.

Reply:
weird， the compiler didn't complain that， anyway， I will update there